src/helpers/pivot/pivot_helpers.ts

```diff
@@ -236,6 +239,9 @@ export function toFunctionPivotValue(
   value: CellValue,
   dimension: Pick<PivotDimension, "type" | "granularity">
 ) {
+  if (value === null) {
+    return `"null"`;
```

Comment:
We already have the issue, and this seems like a very edge case, but the pivots are somewhat broken if there is cells with the string value `null` in the dataset

Reply:
Task to fix in master: https://www.odoo.com/odoo/project/2328/tasks/4063595?cids=1